common/spec/dependabot/pull_request_creator/branch_namer/dependency_group_strategy_spec.rb

```diff
@@ -130,7 +130,7 @@
 
     context "with a grouped security update" do
       let(:directory) { "/" }
-      let(:target_branch) { nil }
+      let(:target_branch) { "" }
```

Comment:
nice!

Reply:
I think in practice this is always a `String`, but a lot of tests take shortcuts, which meant it was `T.nilable(String)`.